2.7/jessie/Dockerfile

```diff
@@ -18,6 +18,7 @@ ENV PYTHONIOENCODING UTF-8
 # extra dependencies (over what buildpack-deps already includes)
 RUN apt-get update && apt-get install -y --no-install-recommends \
 		tk-dev \
+		systemtap-sdt-dev \
```

Comment:
Does this provide any user-facing benefit when we keep it installed?

Reply:
No, that's a miss. Will fix.